meta/SchemaImporter.php

```diff
@@ -72,7 +72,7 @@ public function __construct($table, $json) {
                 $data['new'][] = $column;
             }
         }
-
+        $this->user = $input['user'];
```

Comment:
I guess for your use case it makes sense to be able to set the user via the import data. However for manual imports I would expect that the user running the import is saved here rather than whatever is in the input. Otherwise it's impossible to see who messed up a schema by an import in the planned history feature.